### ISSUE TITLE
[FR] Add a poster for each entry in the tops

### ISSUE THREAD

**Issue by hbooo** [aunefyren/plex-wrapped#12]
Hi, 

It could be great to have a poster of the show or the movies that are in the top : 
![image](https://user-images.githubusercontent.com/242125/144607801-4ea3e10a-7d2b-48dd-affe-1559b944a67e.png)

**Comment by aunefyren** [comment]
I agree. More integration with metadata from outside sources would make for more interesting statistics. Also Tautulli has more data and images to utilize.

**Comment by hbooo** [comment]
Yes could also be nice to have top genre movies/shows/music

**Comment by aunefyren** [comment]
Added in `v3.3.0`.
